wnp-server/src/main/java/com/wonnapark/wnpserver/domain/episode/application/EpisodeManageService.java

```diff
@@ -0,0 +1,93 @@
+package com.wonnapark.wnpserver.domain.episode.application;
+
+import com.wonnapark.wnpserver.domain.episode.Episode;
+import com.wonnapark.wnpserver.domain.episode.dto.request.EpisodeArtistCommentUpdateRequest;
+import com.wonnapark.wnpserver.domain.episode.dto.request.EpisodeCreationRequest;
+import com.wonnapark.wnpserver.domain.episode.dto.request.EpisodeReleaseDateTimeUpdateRequest;
+import com.wonnapark.wnpserver.domain.episode.dto.request.EpisodeThumbnailUpdateRequest;
+import com.wonnapark.wnpserver.domain.episode.dto.request.EpisodeTitleUpdateRequest;
+import com.wonnapark.wnpserver.domain.episode.dto.request.EpisodeUrlsUpdateRequest;
+import com.wonnapark.wnpserver.domain.episode.infrastructure.EpisodeRepository;
+import com.wonnapark.wnpserver.domain.webtoon.Webtoon;
+import com.wonnapark.wnpserver.domain.webtoon.infrastructure.WebtoonRepository;
+import jakarta.persistence.EntityExistsException;
+import jakarta.persistence.EntityNotFoundException;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import static com.wonnapark.wnpserver.domain.episode.application.EpisodeErrorMessage.DUPLICATED_EPISODE;
+import static com.wonnapark.wnpserver.domain.episode.application.EpisodeErrorMessage.EPISODE_NOT_FOUND;
+import static com.wonnapark.wnpserver.domain.episode.application.EpisodeErrorMessage.WEBTOON_NOT_FOUND;
+
+
+@Service
+@RequiredArgsConstructor
+@Transactional(readOnly = true)
+public class EpisodeManageService implements EpisodeManage {
+
+    private final EpisodeRepository episodeRepository;
+    private final WebtoonRepository webtoonRepository;
+
+    @Override
+    @Transactional
+    public Long createEpisode(Long webtoonId, EpisodeCreationRequest request) {
+        if (episodeRepository.existsByTitle(request.title())) {
+            throw new EntityExistsException(String.format(DUPLICATED_EPISODE.getMessage(), request.title()));
```

Comment:
이런 식으로 getMessage 메서드 안에서 에러 메세지를 완전하게 만드는 것도 좋은 방법일 수 있을 것 같아요
```suggestion
            throw new EntityExistsException(DUPLICATED_EPISODE.getMessage(request.title()));
```

Reply:
변경했습니다.